client/src/main/java/io/pravega/client/ClientConfig.java

```diff
@@ -73,6 +75,9 @@ public ClientConfig build() {
                 controllerURI = URI.create("tcp://localhost");
             }
             extractCredentials();
+            if (credentials == null) {
+                log.warn("The credentials are not specified or could not be extracted.");
+            }
```

Comment:
Does this imply that any use of Pravega in an insecure configuration will produce a warning in the client log?

Reply:
Yes. The consensus is that it is good for the user to know that they are not using any creds.